shared/teams/role-picker/index.tsx

```diff
@@ -280,7 +279,10 @@ const Header = () => (
 )
 
 const RolePicker = <IncludeSetIndividually extends boolean>(props: Props<IncludeSetIndividually>) => {
-  const selectedRole = filterRole(props.selectedRole || props.presetRole)
+  const filteredRole = filterRole(props.presetRole)
+  const [selectedRole, setSelectedRole] = React.useState<Role<IncludeSetIndividually>>(
```

Comment:
if presetRole changes due to the parent, we should probably change selectedRole. Maybe a useEffect, or maybe include it in the onOpen